camera_calibration/nodes/cameracalibrator.py

```diff
@@ -71,6 +71,9 @@ def main():
                      help="calibration pattern to detect - 'chessboard', 'circles', 'acircles', 'charuco'\n" +
                      "  if 'charuco' is used, a --charuco_marker_size and --aruco_dict argument must be supplied\n" +
                      "  with each --size and --square argument")
+    group.add_option("--mask",
+                     action="store_true", dest="mask", default=False,
+                     help="Whether displaying or no an overlaying mask displaying corner detection on top of the image")
```

Comment:
Grammar is weird here, 'Whether displaying of no'
I would just change it to 
'Set to display a mask with detected corners on top of the image'